packages/features/bookings/Booker/config.ts

```diff
@@ -54,28 +59,86 @@ export const resizeAnimationConfig: ResizeAnimationConfig = {
   small_calendar: {
     default: {
       width: "calc(var(--booker-meta-width) + var(--booker-main-width))",
+      minHeight: "450px",
+      height: "auto",
       gridTemplateAreas: `"meta main"`,
       gridTemplateColumns: "var(--booker-meta-width) var(--booker-main-width)",
     },
     selecting_time: {
-      width: "100%",
-      maxWidth: "calc(var(--booker-meta-width) + var(--booker-main-width) + var(--booker-timeslots-width))",
+      width: "calc(var(--booker-meta-width) + var(--booker-main-width) + var(--booker-timeslots-width))",
```

Comment:
Keeping maxwidth in instead of width would make awkward fullscreen layout animations. using width instead of maxwidth improves it a lot.